scripts/install/opt/seagate/cortx/motr/conf/support.yaml

```diff
@@ -1,4 +1,2 @@
-
-bundle:
-    create:
-      url: "sh /opt/seagate/cortx/motr/libexec/m0reportbug-bundler"
+support_bundle:
+    - "sh /opt/seagate/cortx/motr/libexec/m0reportbug-bundler"
```

Comment:
Do we have any dependency on CSM for this ? i.e. does this has corresponding changes in CSM as well. 
AFAIK this support.yaml is consumed by CSM only as of now.

Reply:
That will be taken care @yatin-mahajan